docs/general/manage-tokens/holding-rose-tokens/ledger-wallet.md

```diff
@@ -1,14 +1,18 @@
-# Self-Custody With Ledger
+# Self-Custody With Ledger Hardware Wallet
 
-## Install Oasis App
+How to use Ledger hardware wallet with Oasis Network (Oasis app) and Ledger Live software. 
 
-To use your [Ledger](https://www.ledger.com) wallet to hold your ROSE tokens, install the [Oasis app](https://github.com/Zondax/ledger-oasis) on your Ledger wallet via [Ledger Live](https://www.ledger.com/ledger-live/)'s Manager.
+This guide is using Ledger Nano S (Secure Element 2.1.0, Microcontroller 1.12, Bootloaded 0.11), Ledger Live (version 2.40.2) and official [Oasis Wallet - Web](https://wallet.oasisprotocol.org).
 
-:::info
+## Setup your Ledger device and Install Oasis App
 
-Currently, you will have enable _Developer mode_ by going to _Settings -> Experimental features_, to be able to find the Oasis app in Ledger Live's Manager.
+1. To use your [Ledger](https://www.ledger.com) wallet to hold your ROSE tokens, you will have to install [Oasis app](https://github.com/Zondax/ledger-oasis) on your Ledger wallet via [Ledger Live](https://www.ledger.com/ledger-live/)'s Manager. You need to connect your Ledger to your device and unlock it with your PIN code first.
 
-:::
+![](<../../images/wallet/ledger/1_unlock_ledger.png>)
+
+2. Currently, you will have enable _Developer mode_ by going to _Settings -> Experimental features_ -> enable _Developer mode_ toggle, to be able to find the Oasis app in Ledger Live's Manager.
```

Comment:
Strange inherited grammar: "you will have enable"

Reply:
"you will have TO enable" applied.

Reply:
"you will have TO enable" applied.